css/style.css

```diff
@@ -1,3 +1,69 @@
+body{
+  font-family: sans-serif;
+  text-align: center;
+  background-color: lightgray;
+}
+
 h1 {
   color: purple;
+  font-size: 3.5rem;
+}
+
+h2{
+  font-size: 2.5rem;
+  font-weight: 700;
+}
+
+h3, p{
+  color: brown;
+  font-size: 2rem;
+  margin: auto 5px;
+}
+
+#message{
+  left: 20%;
+  color: rgb(233, 127, 77);
+}
+
+#reset{
+  right: 20%;
+  color: rgb(174, 190, 30);
+}
+
+#reset:hover{
+ color: rgb(128, 230, 107);
+}
+
+#numberOfWins{
+  
+}
```

Comment:
Since you didn't end up using #numberOfWins for CSS, you could have commented out or deleted this.  It doesn't cause errors but just worth noting!